### PR TITLE
[OGM] Add Hibernate OGM 5.3 CR1

### PR DESCRIPTION
**Pull request by fax4ever**

### CODE REVIEW

File: ogm/releases/5.3/index.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+=== Core
+
+==== Hibernate ORM 5.2 support
+
+As of Hibernate OGM 5.2, the version of ORM to use with OGM is 5.2 (use the latest micro version).
+
+==== Hibernate Search 5.9 support
+
+As of Hibernate Search 5.9, the version of Search to use with Search is 5.3 (use the latest micro version).
```
Comment (gsmet) [inline ogm/releases/5.3/index.adoc]: This sentence is incorrect.

File: ogm/releases/5.3/index.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+==== Upgrade HQL Parser to 1.5.0.Final
+
+Hibernate Query Parser 1.5.0.Final is now supported.
```
Comment (gsmet) [inline ogm/releases/5.3/index.adoc]: No need to put it there. We bumped the versions because we needed to.

File: ogm/releases/5.3/index.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+=== Datastores
+
+==== Infinispan
+
+ * Infinspan Clustered Counters for sequences generation with Infinispan Embedded are now created at startup
+ * Infinspan Clustered Counters name for id generation also contain the table and not only the columname in InfinispanEmbedded
+
+==== MongoDB
```
Comment (gsmet) [inline ogm/releases/5.3/index.adoc]: If there's nothing to say, don't put anything, let's keep it short.

File: ogm/releases/5.3/index.adoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+==== Upgrade HQL Parser to 1.5.0.Final
+
+Hibernate Query Parser 1.5.0.Final is now supported.
+
+=== Datastores
+
+==== Infinispan
+
+ * Infinspan Clustered Counters for sequences generation with Infinispan Embedded are now created at startup
+ * Infinspan Clustered Counters name for id generation also contain the table and not only the columname in InfinispanEmbedded
```
Comment (gsmet) [inline ogm/releases/5.3/index.adoc]: s/Infinspan/Infinispan/ for the 2 sentences

File: ogm/releases/5.3/index.adoc [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+As of Hibernate Search 5.9, the version of Search to use with Search is 5.3 (use the latest micro version).
+
+==== Upgrade HQL Parser to 1.5.0.Final
+
+Hibernate Query Parser 1.5.0.Final is now supported.
+
+=== Datastores
+
+==== Infinispan
```
Comment (gsmet) [inline ogm/releases/5.3/index.adoc]: I would do it this way:
```
==== Infinispan Embedded

 * The Infinispan Clustered Counters used for sequence generation are now created at startup
 * Their names now also contain the table name and not only the column name
```

File: ogm/releases/5.3/index.adoc [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+ * Infinspan Clustered Counters for sequences generation with Infinispan Embedded are now created at startup
+ * Infinspan Clustered Counters name for id generation also contain the table and not only the columname in InfinispanEmbedded
+
+==== MongoDB
+
+==== Neo4j
+
+==== Integration Test and Provisioning
```
Comment (gsmet) [inline ogm/releases/5.3/index.adoc]: I don't think we need this paragraph. The idea is to be synthetic here and provide information useful for the users. This is internal work.

File: _data/projects/ogm/releases/5.3/series.yml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+summary: Upgrade versions to Hibernate ORM 5.2 and Hibernate Search 5.9
```
Comment (gsmet) [inline _data/projects/ogm/releases/5.3/series.yml]: I would remove `versions`.

File: _data/projects/ogm/releases/5.3/5.3.0.CR1.yml [hunk 8]
```diff
@@ -0,0 +1,7 @@
+version: 5.3.0.CR1
+version_family: 5.3
+date: 2018-02-13
+stable: false
+announcement_url: http://in.relation.to/2018/02/13/hibernate-ogm-5-3-CR1-released/
+summary: Upgrade versions to Hibernate ORM 5.2 and Hibernate Search 5.9
```
Comment (gsmet) [inline _data/projects/ogm/releases/5.3/5.3.0.CR1.yml]: I would remove `versions`.

File: ogm/releases/5.3/index.adoc [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+=== Core
+
+==== Hibernate ORM 5.2 support
+
+As of Hibernate OGM 5.2, the version of ORM to use with OGM is 5.2 (use the latest micro version).
+
+==== Hibernate Search 5.9 support
+
+As of Hibernate Search 5.9, the version of Search to use with Search is 5.9 (use the latest micro version).
+
```
Comment (DavideD) [inline ogm/releases/5.3/index.adoc]: I wouldn't use the sub-title in this core section. Let's keep it consistent with the other releases and section.
I would write the the main purpose of this release is to work with ORM 5.2 and Search 5.9.

File: ogm/releases/5.3/index.adoc [hunk 10]
```diff
@@ -0,0 +1,18 @@
+:awestruct-layout: project-releases-series
+:awestruct-project: ogm
+:awestruct-series_version: "5.3"
+
+=== Core
+
+==== Hibernate ORM 5.2 support
+
+As of Hibernate OGM 5.2, the version of ORM to use with OGM is 5.2 (use the latest micro version).
```
Comment (DavideD) [inline ogm/releases/5.3/index.adoc]: This should be 5.3. Anyway, I would keep this simpler. See other comment for the core section.

**Pull request merged**